src/main/port.ts

```diff
@@ -1,29 +1,22 @@
+import net, { AddressInfo } from "net"
 import logger from "./logger"
-import { createServer, AddressInfo } from "net"
 
-const getNextAvailablePort = () => {
-  logger.debug("getNextAvailablePort() start")
-  const server = createServer()
-  server.unref()
-  return new Promise<number>((resolve, reject) =>
-    server
-      .on('error', (error: any) => reject(error))
-      .on('listening', () => {
-        logger.debug("*** server listening event ***")
-        const _port = (server.address() as AddressInfo).port
-        server.close(() => resolve(_port))
-      })
-      .listen({host: "127.0.0.1", port: 0}))
-}
+// todo: check https://github.com/http-party/node-portfinder ?
 
-export const getFreePort = async () => {
-  logger.debug("getFreePort() start")
-  let freePort: number = null
-  try {
-    freePort = await getNextAvailablePort()
-    logger.debug("got port : " + freePort)
-  } catch(error) {
-    throw("getNextAvailablePort() threw: '" + error + "'")
-  }
-  return freePort
+export async function getFreePort(): Promise<number> {
```

Comment:
Same here, how this is related to Vue->React refactoring?

Reply:
It's not like normal converting single vue->react component :) 
so percept this like almost new app / kick-start..